NMEA2000.h

```diff
@@ -345,6 +349,13 @@ inline void SetN2kProductInformation(tN2kMsg &N2kMsg, unsigned int N2kVersion, u
                   SertificationLevel,LoadEquivalency);
 }
 
+
+void SetN2kPGN59904(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN);
+
+inline void SetN2kPGNISORequest(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN) {
```

Comment:
I like that!   I might use 'Requester' instead of 'Destination' for the 2nd parameter.   Destination sounds to me like our address, not the address of the node making the request.  So:

ool(*ISORequestHandler)(unsigned long RequestedPGN, unsigned char Requester, int DeviceIndex);

A detail:  If the NAK is going to be generated in the NMEA2000.ccp module, It is my understanding the NAK is only sent when the  request was directed to a specific node.  However, for Broadcast Requests (destination address = 0xFF), no such NAK should be sent.